src/main/java/frc/robot/subsystems/intake/BeamBreaker.java

```diff
@@ -0,0 +1,40 @@
+package frc.robot.subsystems.intake;
+
+import edu.wpi.first.wpilibj.DigitalInput;
+import frc.robot.Ports;
+import frc.robot.subsystems.LoggedSubsystem;
+
+
+public class BeamBreaker extends LoggedSubsystem<BeamBreakerLoggedInputs> {
```

Comment:
The beam breaker is not a subsystem

Reply:
well, that's we did it last year

Reply:
No, we didn't. [Link](https://github.com/Galaxia5987/robot-2022/blob/israel-championship/src/main/java/frc/robot/subsystems/conveyor/BeamBreaker.java)

Reply:
Well, gaia wants it that way
https://music.youtube.com/watch?v=LwkrXybZ1uo

Reply:
What is the benefit?